src/FormatNumber/Parser.elm

```diff
@@ -210,6 +245,26 @@ getDecimals locale digits =
         Min min ->
             addZerosToFit min digits
 
+{-| Given a 'NumericSystem` parses a integer `String` into 
+a `List String` representing grouped integers:
+
+    - Western NumericSystem: 1000000 -> 1,000,000
+    - Indian NumericSystem: 1000000 -> 10,00,000
+-}
+
```

Comment:
This is very problematic. Some points to consider:

* There is a typo (a ' instead of a ` NumericSystem, first line of the block)
* The output examples do not look like `List String`
* The input examples do not look like `String`
* There are no tests/examples that run with `elm-verify-tests`
* There are leading whitespace characters at the end of al ine
* There is an extra line between the docs/comment block and the declaration of the function

What about something among these lines?

```suggestion
{-| Given a `NumericSystem` parses a `String` into a `List String` representing
grouped integers:

    splitIntegers Western "1000000" --> ["1", "000", "000"]

    splitIntegers Indian "1000000" -> ["10", "00", "000']

-}
```

Reply:
I will make these changes from my fork as the second test is missing a hyphen.